config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-presubmit-1.26.yaml

```diff
@@ -97,7 +97,7 @@ presubmits:
       extra_refs:
         - org: kubernetes-sigs
           repo: cluster-api-provider-azure
-          base_ref: release-1.7
+          base_ref: release-1.8
```

Comment:
No test run since this change:

- https://testgrid.k8s.io/provider-azure-cloud-provider-azure-1-26-presubmit#pr-cloud-provider-azure-e2e-capz-1-26

Reminder to check back later!